lib/_http_client.js

```diff
@@ -619,6 +612,17 @@ function parserOnIncomingClient(res, shouldKeepAlive) {
 // client
 function responseKeepAlive(req) {
   const socket = req.socket;
+  // There are cases where _handle === null. Avoid those. Passing null to
```

Comment:
```suggestion
  // There are cases where _handle === null. Avoid those. Passing undefined to
```
?

Reply:
Sure, this is just copy paste from the previous location below.